### ISSUE TITLE
Icon should add an identifier to the <svg> element indicating which icon

### ISSUE THREAD

**Issue by sdalonzo** [priceline/design-system#443]
I was working on implementing react-testing-library in relax, and was following a few of the patterns from @jxnblk 's PR: https://github.com/pricelinelabs/design-system/pull/407/files#diff-76b0baeaa20dde93defea3ea13eb463eR108

One thing that is missing on the svg rendered by the `<Icon>` component is a title attribute or some other indicator of what icon it is. This probably also has accessibility concerns, but the primary use case I'm referring to is something like this:
```
const icon = container.querySelector('svg')
expect(icon).toBeTruthy()
```

It would be helpful to be able to verify that the icon rendered is the correct one when writing react-testing-library tests.

**Comment by jxnblk** [comment]
Yeah, I *think* for a11y that "decorative" icons should have the `aria-hidden` attribute and icons with meaning can use the `<title>` tag – not sure how we'll want to handle that, but we should do some research to make sure we handle this in a good way

**Comment by bertya** [comment]
Action item:
- Make `title` prop default to icons name, optionally people can pass in `title`. 
Potential follow up:
- role='img'
- aria-hidden='true'

**Comment by James300** [comment]
Closing since we are making changes to the Icon component for tree-shaking.

**Comment by James300** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Icon should add an identifier to the <svg> element indicating which icon](https://trello.com/c/KeCEdW9R/29-icon-should-add-an-identifier-to-the-svg-element-indicating-which-icon)
